src/libraries/System.Runtime.InteropServices/gen/Common/Resources/xlf/Strings.de.xlf

```diff
@@ -67,16 +67,6 @@
         <target state="translated">Die Basis-COM-Schnittstelle konnte die Quelle nicht generieren. Für diese Schnittstelle wird kein Code generiert.</target>
         <note />
       </trans-unit>
-      <trans-unit id="BaseInterfaceDefinedInOtherAssemblyMessage">
```

Comment:
More removals from changes in the last year: https://github.com/dotnet/runtime/commit/1563fec7ea3e50d27bd6f289bf8a5a9e018c649d#diff-460bca8455e8cb028783e1b1e3f1b9829b0c6a5a2912ee8a6b130786a42696ad